diff --git a/node_modules/snowpack-svgr-plugin/index.js b/node_modules/snowpack-svgr-plugin/index.js
index bc87511..dab346d 100644
--- a/node_modules/snowpack-svgr-plugin/index.js
+++ b/node_modules/snowpack-svgr-plugin/index.js
@@ -18,6 +18,7 @@ module.exports = function (_, options) {
         })
         .replace('import * as React', 'import React');
       const { code: result } = babel.transformSync(code, {
+        filename: path.basename(filePath),
         presets: ['@babel/preset-react'],
       });
       return result;
